prow/pluginhelp/pluginhelp.go

```diff
@@ -57,3 +75,7 @@ type Help struct {
 	PluginHelp         map[string]PluginHelp
 	ExternalPluginHelp map[string]PluginHelp
 }
+
+func (pluginHelp *PluginHelp) addCommand(name string, command Command) {
```

Comment:
This func needs to be capitalized if it is going to be used outside this package.